### PR TITLE
PR to permit winafl to build on windows

### PR DESCRIPTION
**Pull request by gogo2464**
Even with the cicd, I can not compile winafl.

This PR fixed the issue.

### CODE REVIEW

File: winafl.c [hunk 1]
```diff
@@ -455,7 +455,7 @@ instrument_edge_coverage(void *drcontext, void *tag, instrlist_t *bb, instr_t *i
 
     return ret;
 }
-
+#define INOUT
```
Comment (ifratric) [inline winafl.c:458]: Thanks for the patch!

To be on the safe side, I suggest writing this as
#ifndef INOUT
#define INOUT
#endif

For style and visibility reasons, I'd also put it towards the top of the file.

Comment (ifratric) [reply winafl.c:458]: Also, I can't accept a patch unless you sign the CLA.

**Comment by gogo2464** [comment]
sure! let me just sign

**Comment by gogo2464** [comment]
is it signed now?

**Comment by ifratric** [comment]
CLA looks good! Could you update the PR to address the comment?

**Comment by gogo2464** [comment]
yes sure!

**Comment by gogo2464** [comment]
let me do it

**Comment by gogo2464** [comment]
seems ready to merge

**Pull request merged**